### ISSUE TITLE
Defining multiplicity

### ISSUE THREAD

**Issue by LaurensJN** [LaurensJN/IFC#2]
Related to issue #1 ,  I added the IfcSignificanceEnum as attribute to the IfcElement entity. I defined a multiplicity of 0...1:
![image](https://user-images.githubusercontent.com/23737460/108687555-26b3e500-74f7-11eb-8a12-b2d40ceb1488.png)

If I insert it as a template however, the multiplicity is 1...1: 
![image](https://user-images.githubusercontent.com/23737460/108687664-4945fe00-74f7-11eb-9af8-36e2a7a93be5.png)

I don't know how to make the multiplicity 0...1.

**Comment by pjanck** [comment]
Don't do it like that. There's no way for you to change the IFC serialization of IFC files within existing applications / get them to read your files.

Define a new PropertyEnumeration (or sth like that), and connect it to an Element through a PropertySet.

**Comment by LaurensJN** [comment]
Thanks! 

Do you think it is good practice to define a heritage domain where all propertysets are defined? I can't find where to add a PropertySet other than in one of the Core Element / Shared Element / Domain Specific / Resource Definition schemas. See here how I do it now: 

![image](https://user-images.githubusercontent.com/23737460/108977382-c5bc1680-7688-11eb-8e34-93c53469da6a.png)

**Comment by pjanck** [comment]
you're not defining a new schema. that's not possible - you need a broad consensus, international project, etc. 

you need to define property sets with properties in them ... I'm not sure how, but I'll ask ...

**Comment by pjanck** [comment]
you shouldn't be looking in "schema" directories, that is a false positive ...

**Comment by pjanck** [comment]
although, if I think about it - if you're not defining new entities and/or types (that would change the schema), you may as well define new domains ...

**Comment by LaurensJN** [comment]
> you're not defining a new schema. that's not possible - you need a broad consensus, international project, etc.
> 
> you need to define property sets with properties in them ... I'm not sure how, but I'll ask ...

I know that, but there is not really any place else than in one of the schemas to add property sets. Properties, property enumerations and constants however can be added outside the schemas, so I'm doing that. Also I figured that it would be better to create a new -currently non-existent- domain instead of using one of the existing ones

**Comment by pjanck** [comment]
Right, I got you. As long as you don't define new enumeration types or entities (that is, EXPRESS stuff), you should be good to go.
